msg/templates/urtps/microRTPS_agent_CMakeLists.txt.em

```diff
@@ -38,15 +38,15 @@ project(micrortps_agent)
 find_package(fastrtps REQUIRED)
 find_package(fastcdr REQUIRED)
 
-# Set C++11
+# Set C++14
 include(CheckCXXCompilerFlag)
 if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_COMPILER_IS_CLANG OR
     CMAKE_CXX_COMPILER_ID MATCHES "Clang")
-    check_cxx_compiler_flag(--std=c++11 SUPPORTS_CXX11)
+    check_cxx_compiler_flag(--std=c++14 SUPPORTS_CXX14)
     if(SUPPORTS_CXX11)
```

Comment:
@dagar shouldn't have this been changed to `SUPPORTS_CXX14` as well?

Reply:
Yes I suppose so, although the agent is actually a separate build. Care to open a PR fixing it?

Good catch.

Reply:
Sure